wp-admin/js/widgets/media-image-widget.js

```diff
@@ -195,6 +195,19 @@
 
 			mediaFrame.open();
 
+			/*
+			 * Make sure focus is set inside of modal so that hitting Esc will close
+			 * the modal and not inadvertently cause the widget to collapse in the
+			 * customizer.
+			 */
+			mediaFrameContentView = mediaFrame.views.get( '.media-frame-content' )[0];
+			if ( mediaFrameContentView.model && mediaFrameContentView.model.dfd ) {
+				mediaFrameContentView.model.dfd.done( function() {
+					_.defer( function() { // Next tick.
+						mediaFrameContentView.$el.find( '[data-setting="caption"]:first' ).focus();
+					} );
+				} );
+			}
```

Comment:
This chunk I added seems now out of place for this PR and it could be good to remove. Auto-focusing was addressed in another PR and it also seems like an issue the media JS should tackle anyway.